Dapper.EntityFrameworkCore/NetTopologySuiteGeometryHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Data;
+using System.Data.SqlClient;
+using NetTopologySuite.Geometries;
+using NetTopologySuite.IO;
+
+namespace Dapper.EntityFrameworkCore
+{
+    /// <summary>
+    /// Type-handler for the Geometry spatial type.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public class NetTopologySuiteGeometryHandler<T> : SqlMapper.TypeHandler<T>
+        where T : Geometry
+    {
+        readonly bool _geography;
+        readonly SqlServerBytesWriter _writer;
+        readonly SqlServerBytesReader _reader;
+
+        /// <summary>
+        /// Create a new handler instance.
+        /// </summary>
+        /// <param name="geography"></param>
+        public NetTopologySuiteGeometryHandler(bool geography = false)

```

Comment:
something about this bool parameter is making me itch a little; I wonder if there's an alternative - for example, taking an enum or even the `string` (udt-type-name)? or maybe a common abstract base type `NetTopologySuiteHandler<T>` that has an `abstract` type name property, and two concrete sub-types (that could presumably be `internal`): `NetTopologySuiteGeometryHandler<T>` and `NetTopologySuiteGeographyHandler<T>`?

Reply:
I will change to `NetTopologySuiteGeometryHandler<T>` and `NetTopologySuiteGeographyHandler<T>`